README.md

```diff
@@ -11,86 +11,121 @@ You may also use it standalone for your projects.
+*	Module `cmdutils`
+
+  * [x] utilities to work with CLIs
+
+*	Module `conv`
 
   * [x] convert between values and pointers for any types
   * [x] convert from string to builtin types (wraps `strconv`)
+  * [x] require `./typeutils` (test dependency)
 
-*	Package `fileutils`
+*	Module `fileutils`
 
   * [x] file upload type
   * [x] search in path (deprecated)
-* Package `jsonname`
+
+* Module `jsonname`
 
   * [x] infer JSON names from go properties
 
-* Package `jsonutils`
+* Module `jsonutils`
 
   * [x] fast json concatenation
   * [x] read and write JSON from and to dynamic go data structures
+  * [x] require `github.com/mailru/easyjson`
 
-* Package `loading`
+* Module `loading`
 
   * [x] load from file or http
+  * [x] require `./yamlutils`
 
-* Package `mangling`
+* Module `mangling`
 
   * [x] name mangling for go
 
-* Package `netutils`
+* Module `netutils`
 
   * [x] host, port from address
 
-*	Package `stringutils`
+*	Module `stringutils`
 
   * [x] search in slice (with case-insensitive)
-  * [x] slit/join query parameters as arrays
+  * [x] split/join query parameters as arrays
 
-*	Package `typeutils`
+*	Module `typeutils`
 
   * [x] check the zero value for any type
 
-* Package `yamlutils`
+* Module `yamlutils`
 
   * [x] converting YAML to JSON
   * [x] loading YAML into a dynamic YAML document
+  * [x] require `./jsonutils`
+  * [x] require `github.com/mailru/easyjson`
+  * [x] require `gopkg.in/yaml.v3`
 
 ---
 
-This repo has a few dependencies outside of the standard library:
+The root module `github.com/go-openapi/swag` at the repo level maintains a few 
+dependencies outside of the standard library:
 
 * YAML utilities depend on `gopkg.in/yaml.v3`
 * JSON utilities `github.com/mailru/easyjson`
 
+This is not necessarily the case for all sub-modules.
+
 ## Release notes
 
 ### v0.24.0
 
 With this release, we have largely modernized the API of `swag`:
-* The traditional `swag` API still works the same: code that imports `swag` will still
+
+* The traditional `swag` API is still supported: code that imports `swag` will still
   compile and work the same.
 * A deprecation notice is published to encourage consumers of this library to adopt
-  the newer API:
+  the newer API
+* **Deprecation notice** 
   * configuration through global variables is now deprecated, in favor of options passed as parameters
-  * helper functions are moved to more specialized packages, which are exposed as 
+  * all helper functions are moved to more specialized packages, which are exposed as 
     go modules. Importing such a module would reduce the footprint of dependencies.
+  * _all_ functions, variables, constants exposed by the deprecated API have now moved, so
+    that consumers of the new API no longer need to import github.com/go-openapi/swag, but
+    should import the desired sub-module(s).
+
+
+#### Note to contributors
+
+The usual `go test ./...` command, run from the root of this repo won't work any longer to test all submodules.
```

Comment:
FYI `go test work` will solve this in Go 1.25

- https://github.com/golang/go/issues/50745#issuecomment-2730714430

But I know swag still supports Go 1.20... so...

Reply:
I know. I've see you're following that issue. Notice that "go test work" will _partially_ solve this issue, since the go team seems still rather opposed to support the concept of mono-repos.

Reply:
I've left this comment specifically for you... :)